data_model/src/account.rs

```diff
@@ -237,20 +235,18 @@ declare_item! {
     #[display(fmt = "({id})")] // TODO: Add more?
     pub struct Account {
         /// An Identification of the [`Account`].
-        id: Id,
+        pub id: Id,
```

Comment:
So if I understand it correctly, the public fields are always going to be available, so `Account` pointers have a transparent section, right?

Reply:
will be transparent only for `iroha_core`. Macro will make them public fields `pub(crate)` by default unless you activate `transparent_api`